packages/core/src/property.spec.ts

```diff
@@ -192,32 +172,20 @@ describe('combine', () => {
 
 		expect(cb1).toBeCalledTimes(1)
 	})
-	it('multicasts the result observable', () => {
+	it("emits do not triggers updates to the value", () => {
 		const emitter = newEmitter()
 		// we need a property that always has a new value
 		// so that projection function is not memoized by combine
 		const a = newProperty(now, emitter.subscribe)
+
 		const cb = jest.fn(now)
-		const c = combine(a, cb)
+		const c = combine(a, now)
+
 		const cb1 = jest.fn(constVoid)
-		const cb2 = jest.fn(constVoid)
 		c.subscribe({ next: cb1 })
-		c.subscribe({ next: cb2 })
-		// imitate consumer to warm up the cache
-		c.get()
-		cb.mockClear()
+
 		emitter.next(now())
-		expect(cb).toHaveBeenCalledTimes(1)
-	})
-	it('correctly caches undefined value', () => {
```

Comment:
This test should be replicated for the interop observable

Reply:
Moved to interop observable